### DISCUSSION TITLE
ch2-cgo/ch2-08-class

### DISCUSSION THREAD

**Discussion by giscus** [chai2010/advanced-go-programming-book#638] [General]
# ch2-cgo/ch2-08-class

Go语言高级编程

https://chai2010.cn/advanced-go-programming-book/ch2-cgo/ch2-08-class.html

**Comment by ZzzJing** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。
